File: junsuk/18주차/최소 직사각형.js
function solution(sizes) {
    let maxWidth = 0;
    let maxHeight = 0;

    sizes.forEach(([w, h]) => {
        const currentW = Math.max(w, h);
        const currentH = Math.min(w, h);

        if (currentW > maxWidth) maxWidth = currentW;
        if (currentH > maxHeight) maxHeight = currentH;
    });

    return maxWidth * maxHeight;
}

File: jungki/21주차/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    for i in range(len(sizes)):
        if sizes[i][0]>sizes[i][1]:
            sizes[i][0],sizes[i][1]=sizes[i][1],sizes[i][0]
    a=max(sizes)
    a=a[0]
    b=sizes[0][1]
    for j in range(len(sizes)):
        if b<sizes[j][1]:
            b=sizes[j][1]
    total=a*b
    return total